src/modules/blocks/tickets/template.js

```diff
@@ -38,38 +42,41 @@ class Tickets extends PureComponent {
 
 	renderBlock() {
 		const {
-				  isSelected,
-				  clientId,
-				  canCreateTickets
-			  } = this.props;
+			isSelected,
+			clientId,
+			canCreateTickets,
+		} = this.props;
 
 		return (
 			<Fragment>
-				<TicketsContainer isSelected={isSelected}/>
-				{canCreateTickets && <TicketsDashboard isSelected={isSelected} clientId={clientId}/>}
-				<TicketControls/>
+				<TicketsContainer isSelected={ isSelected } />
+				{ canCreateTickets && <TicketsDashboard isSelected={ isSelected } clientId={ clientId } />}
```

Comment:
We should add a space before the ending bracket, too.
```suggestion
				{ canCreateTickets && <TicketsDashboard isSelected={ isSelected } clientId={ clientId } /> }
```